static/src/javascripts/projects/common/modules/experiments/tests/adblocking-response.js

```diff
@@ -0,0 +1,156 @@
+define([
+    'common/utils/$',
+    'common/utils/config',
+    'common/utils/detect',
+    'common/utils/storage',
+    'lodash/collections/contains',
+    'common/modules/commercial/user-features',
+    'common/modules/commercial/survey/survey-adblock',
+    'common/utils/cookies'
+], function (
+    $,
+    config,
+    detect,
+    storage,
+    contains,
+    userFeatures,
+    SurveyAdBlock,
+    cookies
+) {
+    return function () {
+        this.id = 'AdBlockingResponse';
+        this.start = '2016-08-17';
+        this.expiry = '2016-10-18';
+        this.author = 'Justin Pinner';
+        this.description = 'Adblocking response ZERO PERCENT TEST WITH 10 MINUTES GRACE';
+        this.audience = 0;
+        this.audienceOffset = 0;
+        this.successMeasure = 'Adblocking users will either pay, prove they have a subscription or turn off their adblocker.';
+        this.audienceCriteria = 'All non-Firefox users with active adblockers.';
+        var variantDataLinkNames = [
+            ['adblock whitelist no-close'],
+            ['adblock membership no-close'],
+            ['subscriber number no-close'],
+            ['user help email no-close'],
+            ['adblock whitelist immediate-close'],
+            ['adblock membership immediate-close'],
+            ['subscriber number immediate-close'],
+            ['user help email immediate-close'],
+            ['adblock whitelist delayed-close'],
+            ['adblock membership delayed-close'],
+            ['subscriber number delayed-close'],
+            ['user help email delayed-close']
+        ].map(function(_) {
+            return _[0];
+        }).join(', ');
+        this.dataLinkNames = 'adblock response overlay: '+variantDataLinkNames+', subscriber number page: user help email';
+        this.idealOutcome = 'Adblocking users will either pay or allow ads.';
+
+        this.canRun = function () {
+            return !contains('firefox', detect.getUserAgent.browser.toLowerCase())
+                && contains(['desktop', 'leftCol', 'wide'], detect.getBreakpoint())
+                && config.page.edition === 'UK'
+                && !cookies.get('gu_abm_x');
+        };
+
+        this.variants = [{
+            id: 'no-close',
```

Comment:
I wonder if this is due to the vagaries of ad blocker detection. the "no control group" will contain lots of false negatives and thus their behaviour is not a safe baseline.